app/src/main/java/org/oppia/app/player/state/StateFragment.kt

```diff
@@ -7,11 +7,27 @@ import android.view.View
 import android.view.ViewGroup
 import org.oppia.app.fragment.InjectableFragment
 import org.oppia.app.player.audio.CellularDataInterface
+import org.oppia.app.player.exploration.EXPLORATION_ACTIVITY_TOPIC_ID_ARGUMENT_KEY
 import javax.inject.Inject
 
 /** Fragment that represents the current state of an exploration. */
 class StateFragment : InjectableFragment(), CellularDataInterface {
 
+  companion object {
+    /**
+     * Creates a new instance of a StateFragment
+     * @param topicId
+     * @return [StateFragment]: Fragment
+     */
+    fun newInstance(topicId: String): StateFragment {
```

Comment:
Why would we need the topic ID for creating new state fragments? Shouldn't the exploration ID be sufficient?

Reply:
changed to explorationId

Reply:
Please also rename the variable here to be ``explorationId`` to match the KDoc & the key name.